.github/workflows/regression.yml

```diff
@@ -0,0 +1,123 @@
+name: 'regression'
+
+on:
```

Comment:
Can you add a comment that this is meant for on-demand triggering ?

Reply:
Renamed the file and the workflow itself.